src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java

```diff
@@ -20,22 +20,22 @@ public class InputRequireThisTryWithResourcesOnlyOverlappingFalse implements Aut
 
     void oneResource() {
         try (BufferedReader fldBufferedReader
-                     = new BufferedReader(new InputStreamReader(null, "utf-8"))) { } // ok
```

Comment:
@abhishekmaity ,
It is required to remove the respective suppressions from the file [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)

Please look at previously merged PRs for examples of update : 
- https://github.com/checkstyle/checkstyle/pull/14258
- https://github.com/checkstyle/checkstyle/pull/14259
- https://github.com/checkstyle/checkstyle/pull/14260

Reply:
@MANISH-K-07 Thanks, corresponding changes made at .xml file